R/R2R/bundled/patches/fix-format-strings.patch

```diff
@@ -0,0 +1,68 @@
+diff --git a/src/ParseOneStockholm.cpp b/src/ParseOneStockholm.cpp
```

Comment:
Any chance to upstream these patches?  As far as I can tell they can be useful to everyone else?

Reply:
I will send the original author an email with the patches.